src/android/patches/makeBuildPatch.js

```diff
@@ -1,6 +1,8 @@
 module.exports = function makeBuildPatch(name) {
   return {
     pattern: /[^ \t]dependencies {\n/,
-    patch: `    compile project(':${name}')\n`,
+    patch: `    compile project(':${name}') {
+        exclude group: 'com.facebook.react', module: 'react-native'
```

Comment:
Unlinking wise, shouldn't we handle both cases? (to be backwards compatible) Or isn't that changing anything?